### PR TITLE
Bf 88 eligibility criteria

### PR DESCRIPTION
**Pull request by shewood**
# Description

[BF-88](lhttps://bf-decd.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=BF&modal=detail&selectedIssue=BF-88)

Added sans serif for font. 
Added Criteria Grid (can be removed or combined with existing Grid later once more details are known regarding the layout from designers and what is needed to be shown)
Added Criteria Box component that can contain one of:
 - Added Select picker component
 - Added Number box component

## Meets Definition of Done
- [ x] Unit tests up to date
- [ ] E2E tests up to date 
- [ x] I18n

### CODE REVIEW

File: components/organisms/CriteriaGrid.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import PropTypes from "prop-types";
+import { CriteriaBox } from "../atoms/CriteriaBox";
+import { SelectPicker } from "../atoms/SelectPicker";
+import { NumInput } from "../atoms/NumInput";
+
+import { useTranslation } from "next-i18next";
+
+export function CriteriaGrid(props) {
```
Comment (var-kyle) [inline components/organisms/CriteriaGrid.js:8]: This grid looks great! My only concern is that it's essentially hardcoded with the three CriteriaBoxes and none of the props being passed in are being used.

**Review by var-kyle — CHANGES_REQUESTED** [review]

File: components/atoms/CriteriaBox.stories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  component: CriteriaBox,
+};
+
+const Template = (args) => <CriteriaBox {...args} />;
+
+export const Primary = Template.bind({});
+
+Primary.args = {
+  criteriaTitle: "Title Text",
+};
```
[the remark this answers is not in the document]
Comment (shewood) [reply components/atoms/CriteriaBox.stories.js:15]: I think what you are looking for here is found in this component?? https://dts-stn.github.io/Benefits-Finder/refs/pull/53/merge/?path=/story/components-organisms-criteriagrid--primary

File: components/atoms/CriteriaBox.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+export function CriteriaBox(props) {
+  return (
+    <div className="block flex-1 my-1 mx-1 p-4 md:my-0 md:mx-0 rounded-lg shadow-lg border border-gray-200 ">
+      {/* <label
+        className="text-gray-700 font-display font-bold"
+        data-cy={props.dataCy}
+      >
+        {props.criteriaTitle}
+      </label> */}
```
Comment (var-kyle) [inline components/atoms/CriteriaBox.js:11]: Please remove the commented out `<label>`

File: components/atoms/NumInput.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      <input
+        type="number"
+        min="1"
+        step="1"
+        max="120"
+        className="form-input my-2  block w-full rounded"
+        placeholder={props.placeholder}
+        data-cy={props.dataCy}
+      ></input>
+    </label>
```
Comment (var-kyle) [inline components/atoms/NumInput.js:19]: Awesome! This should be more reusable with the label already there.

File: components/atoms/NumInput.stories.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  component: NumInput,
+};
+
+const Template = (args) => <NumInput {...args} />;
+
+export const Primary = Template.bind({});
+
+Primary.args = {
+  placeholder: "Placeholder Text",
+};
```
Comment (var-kyle) [inline components/atoms/NumInput.stories.js:15]: Just forgot to add the `criteriaTitle` prop here

File: components/atoms/SelectPicker.stories.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  name: "Name",
+  selects: [
+    {
+      criteriaSelect: "Ontario",
+    },
+    {
+      criteriaSelect: "Alberta",
+    },
+  ],
+};
```
Comment (var-kyle) [inline components/atoms/SelectPicker.stories.js:24]: Forgot to add `criteriaTitle` to the args

File: components/organisms/CriteriaGrid.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+
+import { useTranslation } from "next-i18next";
+
+export function CriteriaGrid(props) {
+  const { t } = useTranslation("common");
+  return (
+    <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-10 px-2  md:px-0">
+      {props.children}
+    </div>
```
Comment (var-kyle) [inline components/organisms/CriteriaGrid.js:10]: I know this doesn't look like much here, but the grid really is better than the other one!

File: pages/index.js [hunk 8]
```diff
@@ -77,6 +81,88 @@ export default function Home({
         <CardGrid>{categories}</CardGrid>
       </section>
 
+      {/* your situation section */}
+      <section id="eligibility_criteria" className="">
+        <h3 className="text-2xl text-bold py-3">{t("eligibilityCriteria")}</h3>
+        <CriteriaGrid>
+          {/* location picker */}
+          <CriteriaBox>
+            <SelectPicker
```
Comment (var-kyle) [inline pages/index.js:90]: Looks messy, but it's perfect for now, until we can figure out how we're going to handle this.

**Review by var-kyle — CHANGES_REQUESTED** [review]
A lot of good improvements, just a couple of small changes and it's good to merge!

File: components/atoms/CriteriaBox.js (hunk 3 above)
Comment (shewood) [reply components/atoms/CriteriaBox.js:11]: oops, I took this out then added it back to see where it was breaking. Forgot to take it back out again. It's gone now.

**Review by var-kyle — APPROVED** [review]
Awesome work! Looks great!

**Pull request merged**